src/scales.js

```diff
@@ -327,6 +327,7 @@ export function scaleOrder({range, domain = range}) {
 // example, a rect will span the full extent of the chart along a collapsed
 // dimension (whereas a dot will simply be drawn in the center).
 export function isCollapsed(scale) {
+  if (!scale || !scale.domain) return true;
```

Comment:
If scale is truthy, would scale.domain ever be falsey? Can we do this instead:

```suggestion
  if (scale === undefined) return true;
```